### ISSUE TITLE
Does the submission of pytorch models require "logit" layer in them?

### ISSUE THREAD

**Issue by chengxuz** [brain-score/sample-model-submission#16]
I have prepared my models following the example AlexNet script. It seems to pass most of the tests until the logit behavior one, according to which the length of the "neuroid" is not at 1000 (error message from [here](https://github.com/brain-score/model-tools/blob/master/model_tools/brain_transformation/behavior.py#L45)). The line above this line seems to extract the responses at the layer "logits", which would not work for my models at all and I also don't think would work for a lot of models, as they don't have the layer in this exact name. Is it possible to remove this check? Does this failure mean that the model cannot be submitted to Brain-score?

Best,
Chengxu
